packages/paste-website/package.json

```diff
@@ -29,6 +29,7 @@
     "@twilio-paste/button": "^2.3.6",
     "@twilio-paste/combobox": "^1.3.18",
     "@twilio-paste/combobox-primitive": "^0.1.6",
+    "@twilio-paste/core": "^2.16.51",
```

Comment:
Should we drop the other Paste packages?

Reply:
Hard to say. On the one hand it's a nice testing ground to make sure the individual components still work, along side core. On the other it's kind of a mess.

I think the ideal is in another PR we swap to using core everywhere, using the unbarrelled imports so we're still testing the isolated components. And We have a bundle size check on the website to ensure we don't regress anything in the future.